anatomy/tracks/concept-exercises.md

```diff
@@ -450,6 +450,45 @@ Some languages require additional files for the tests to run. Example of these a
 
 Some files are not specific to individual exercises, but are instead applicable to _all_ exercises. Check the [documentation](./shared-files.md) for more information.
 
+## Naming
+
+Concept Exercises should be named after their story/theme, _not_ after their concept(s).
+
+Good examples of names:
+
+- `Tim from Marketing`
+- `Lucian's Luscious Lasagna`
+- `Calculator Conundrum`
+
+Bad examples of names:
+
+- `Booleans`: should not be named after a concept
+- `Exercise #1`: an exercise is not a story/theme
+
+When forking an exercise without major changes, use the original name when possible.
+
+### Slugs
+
+Each exercise also has a _slug_, which is normalized version of the exercise name using the following rules:
+
+1. Use lowercase.
+1. Use [kebab-case][kebab-case].
+1. Use alphanumerical characters ('a' - 'z') and dashes ('-').
+
+Good examples of slugs:
+
+- `tim-from-marketing`
+- `lucians-luscious-lasagna`
+- `calculator-conundrum`
+
+Bad examples of slugs:
+
+- `TIM-FROM-MARKETING`: should use lowercase (i.e. `tim-from-marketing`)
+- `TimFromMarketing`: should use kebab-case (i.e. `tim-from-marketing`)
+- `floating-point-numbers`: should not use a concept
```

Comment:
*Must* the slugs follow the rules above or *should* they follow those rules?

The paragraph above suggests they must follow those rules, but these examples suggest they're optional.

Reply:
must.


```suggestion
Bad examples of slugs:

- `TIM-FROM-MARKETING`: does not use lowercase (i.e. `tim-from-marketing`)
- `TimFromMarketing`: does not use kebab-case (i.e. `tim-from-marketing`)
- `floating-point-numbers`: uses a concept name not a exercise
```

Reply:
https://github.com/exercism/docs/pull/78